app/admin/books_admin.rb

```diff
@@ -0,0 +1,41 @@
+Trestle.resource(:books) do
+  menu do
+    item :books, icon: "fa fa-star"
+  end
+
+  # Customize the table columns shown on the index view.
+  #
+  # table do
+  #   column :id
+  #   column :title
+  #   column :price
+  #   column Subject.find( :subject_id ).name
+  #   column :created_at, align: :center
+  #   # actions
+  # end
+
+  # Customize the form fields shown on the new/edit views.
+  #
+  form do |book|
+    text_field :title
+    text_field :price
+    subject_select = Subject.all
+    select :subject_id, subject_select
+    text_field :description
+  
+    row do
+      col { datetime_field :created_at }
+    end
```

Comment:
I wouldn't let the created_at field be editable, Rails takes care of this for you. You can remove this field